.github/workflows/pr-build-live-branch.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Live Branch
+on:
+  push:
```

Comment:
We don't need on push as well, it creates two jobs

<img width="922" alt="image" src="https://user-images.githubusercontent.com/1922453/201818380-b65b648d-951d-4448-a24a-76befa8e9318.png">